src/dotnet/CarbonAware.CLI/Program.cs

```diff
@@ -1,14 +1,42 @@
-﻿namespace CarbonAwareCLI;
+﻿namespace CarbonAware.CLI;
+
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.Aggregators.Configuration;
+using Microsoft.Extensions.DependencyInjection;
 
 class Program
 {
-    static void Main(string[] args)
-    {
-        var cli = new CarbonAwareCLI(args);
+    private static ICarbonAwareAggregator _aggregator;
+    public static void Main(string[] args)
```

Comment:
you could defined `Main` as `static async Task<int> Main(string[] args)`, so you don't need to do `emissions.Result`. As a general comment, mixing sync with async is not recommended.